main.go

```diff
@@ -81,6 +82,13 @@ func main() {
 		lineLength := len(data)
 		var output [][]byte
 
+		if lineLength < 2 && *onlyDelimitParameter {
```

Comment:
Please explain this condition.

Reply:
well the second one is not hard to understand and the first one is there to check if there are several values in one line separated by a delimiter

Reply:
Which several values? Shouldn't you check for the delimiter?

https://github.com/Al2Klimov/trainee-go-cut/pull/13/commits/a0d402f8bd778160a43c6de8a6a69cea4f523156#diff-7ddfb3e035b42cd70649cc33393fe32cR19